mmv1/third_party/terraform/acctest/tgc_utils.go

```diff
@@ -1,64 +1,307 @@
 package acctest
 
 import (
+	"encoding/base64"
+	"encoding/json"
 	"fmt"
 	"log"
 	"regexp"
+	"runtime"
 	"strings"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/terraform"
 )
 
+type ResourceMetadata struct {
+	CaiAssetName    string         `json:"cai_asset_name"`
+	ResourceType    string         `json:"resource_type"`
```

Comment:
Do we want to keep `ResourceType` with `ResourceAddress`?

Reply:
Personally I'm inclined to keep it. I realize the information can be retrieved through the data, but it's nice to have for ease of access and not harmful? What do you think?

Reply:
Then we can keep it.